.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 8
+  - 9
```

Comment:
This is the nodejs version used to execute tests so not related to the typings version. Not sure if this is really intended to step to NodeJs 9 here. fyi @andy-ms

Reply:
Yeah, this should be done in a separate PR if it is done. I brought it down to 8 when the new release started causing trouble, though it may be fixed now. (#21156)

Reply:
build was executing fine with node 9